### DISCUSSION TITLE
No mouse control... something with VNC?

### DISCUSSION THREAD

**Discussion by cchobanIL** [jlesage/docker-firefox#252] [Q&A]
Ok, so first and foremost, this is brilliant.  Thank you.

Second, I am by no means an engineer-level anything, and in many cases can just sort of get by getting things running.  Let's call me a very technical hobbyist.  With that in mind, I have the container running on a Synology NAS, and can access it successfully through a reverse proxy.  So, it's fine hitting it from, say, "firefox.mydomain.com".  I enter credentials, and I see the google home page, because that's what I have the opening URL environment variable set to.

But that's where it gets puzzling.  The ONLY thing I have mouse access to, is the tab outlined in red in the picture below.  Once my mouse ventures outside that box, there is no mouse control.  Now, it appears to be because it's still part of my local OS... I **DO** see a black mouse cursor in the middle of the screen (black because I've got it set to dark mode), but nothing I do can move it.

Does this have something to do with VNC?  Because VNC is a remote control/remote viewing app.  I have opened and directed VNC port 5900 to my Synology, port 5900, and I've even tried setting up a second reverse proxy entry to direct 5900 to the container.  (Grasping beyond my know-how.)  Any help?

![image](https://github.com/user-attachments/assets/0667bb72-39ab-4373-ab91-2e4d97e92391)

**Comment by jlesage** [comment]
If you close the side menu outlined in red, clicking the Firefox interface does noting ?
Did you try to access the container directly, without going through the reverse proxy ?
Did you try to access the container with another browser ?
